### DISCUSSION TITLE
求教为什么nb run启动之后端口会不通呢

### DISCUSSION THREAD

**Discussion by w125939224** [nonebot/discussions#334] [问答] [answered]
![3bf7bfc6b31b5c47e1dcffad1c09356](https://user-images.githubusercontent.com/68673870/222373831-37b788b4-c36f-4057-86d4-3215551e1c96.png)
配置如下：
![1144a9b9a01707e4aaee375aa1540cf](https://user-images.githubusercontent.com/68673870/222373929-fcc05542-6df3-42eb-b550-e1f96b305888.png)
![5faff1e9b6501498f9d1e7305019e84](https://user-images.githubusercontent.com/68673870/222373943-54251ad0-dd2d-4a8b-be92-2c450d4b840e.png)

**Comment by yanyongyu** [comment, marked as answer]
别把nb关掉啊，开着才能连

**Reply by w125939224** [reply]
是开着的，是后面才关掉的，看了一下防火墙

**Reply by w125939224** [reply]
感谢回复，速度真快~但是依然需要请教。
![d17698ac70915942d64aaa65fd3404e](https://user-images.githubusercontent.com/68673870/222377647-b7f314a1-380e-4806-9574-70c57085d9e5.png)
![60c11bacf7721e76e8fe84f5b2fc5ec](https://user-images.githubusercontent.com/68673870/222377780-ec669471-d39f-4e58-97c6-219bf0ccdb59.png)

**Reply by yanyongyu** [reply]
你使用的driver并不支持监听端口，你需要改成`DRIVER=~fastapi`

**Reply by w125939224** [reply]
改用fastapi之后提示如下，请求帮助
![94f349db13339c24dd0ec1db88a65d3](https://user-images.githubusercontent.com/68673870/222604495-a3c54275-029c-43c1-9271-62bae5edab88.png)

**Reply by w125939224** [reply]
再次修改.env.prod配置文件解决了。感谢您回复~

**Reply by QihangYang** [reply]
@w125939224 你好！我也遇到了同样的问题，但是我的.env.prod文件里已经写了DRIVER=~fastapi了，还是打不开。请问可以看一下你的配置文件是怎么写的吗？

**Reply by yanyongyu** [reply]
@QihangYang 你可以日志截图

**Reply by QihangYang** [reply]
@yanyongyu 不好意思我第一次用，你说的日志截图指的是这个吗？
<img width="828" alt="截屏2023-03-03 22 30 29" src="https://user-images.githubusercontent.com/48112998/222746993-9ff3ca72-0a15-48a7-839c-8017a75be663.png">

**Reply by yanyongyu** [reply]
日志里并没有看见你的gocq连接

**Reply by QihangYang** [reply]
@yanyongyu 不好意思，请问可以麻烦再稍微多解释一点吗？我是在macOS的系统上，pycharm里按照https://zhuanlan.zhihu.com/p/600607068 这个文档进行的操作，进行到“进入浏览器进入http://127.0.0.1:8080/go-cqhttp/ ”时，打开浏览器只有“{"detail":"Not Found"}”。请问是哪里有什么问题吗？

**Reply by yanyongyu** [reply]
看上去你没有加载适配器也没有安装gocq插件

**Reply by yanyongyu** [reply]
```bash
nb adapter install "OneBot V11"
nb plugin install nonebot-plugin-gocqhttp
```

**Reply by QihangYang** [reply]
我还是没有成功，应该是环境的设置上哪里存在一些问题。我再去多查阅一些资料，重头再试一次吧。感谢你的回答！

**Reply by w125939224** [reply]
![61caf980809d83dd1abb8470c0c2909](https://user-images.githubusercontent.com/68673870/222998450-95ceb582-949d-4cf5-a87e-274bfce7497e.png)
